<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisasterDialog</class>
 <widget class="QDialog" name="DisasterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_type">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_duration">
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_size">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_rate">
            <property name="text">
             <string>Rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_target">
            <property name="text">
             <string>Target</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox_disaster">
            <item>
             <property name="text">
              <string>Inferno</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Meteor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gamma Ray</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Drought</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Annihilation</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_duration">
            <property name="maximum">
             <number>2000</number>
            </property>
            <property name="singleStep">
             <number>250</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_size">
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>800</number>
            </property>
            <property name="singleStep">
             <number>25</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_rate">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_target">
            <item>
             <property name="text">
              <string>Land Dino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Air Dino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Water Dino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tree</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bush</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Grass</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alga</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>216</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DisasterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DisasterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
